views/incorrect-information/wrong-ro.html

```diff
@@ -1,23 +1,101 @@
 {% extends "layout.html" %}
 {% block pageTitle %}
-  Incorrect RO
+  Incorrect registered office address - File a confirmation statement
 {% endblock %}
+
 {% block backLink %}
   {% include "includes/back-link.html" %}
 {% endblock %}
+
 {% block content %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <form action="" method="post">
-        <h1 class="govuk-heading-xl">
+        <h1 class="govuk-heading-l">
           You need to update the company details
         </h1>
-        <p>You will need to
-          <a href= "{{changeRoaUrl}}" target="_blank" rel="noopener noreferrer" data-event-id="update-roa-link">update the registered office address (opens in a new tab)</a>
-          before you can file a confirmation statement.</p>
-        {{ govukInsetText({
-            text: "An update to the registered office address can take some time to process, so it may not be reflected immediately in the information displayed on this service. As long as the update has been submitted, you can still file the confirmation statement."
-        }) }}
+
+        <ol class="app-step-nav__steps">
+          <li class="app-step-nav__step js-step step-is-shown">
+            <div class="app-step-nav__header" data-position="1">
+              <h2 class="app-step-nav__title">
+                <span class="app-step-nav__circle app-step-nav__circle--number">
+                  <span class="app-step-nav__circle-inner">
+                    <span class="app-step-nav__circle-background">
+                      <span class="govuk-visually-hidden">Step</span> 1
+                    </span>
+                  </span>
+                </span>
+                <span class="js-step-title">
+                  <button class="app-step-nav__button app-step-nav__button--title js-step-title-button"/>
+                </span>
+                <span class="js-step-title-text">
+                  Update your registered office address
+                </span>
+              </h2>
+            </div>
+            <div>
+              <ol class="app-step-nav__list " data-length="1">
+                <li class="app-step-nav__list-item js-list-item">
+                  <p>You will need to
+                    <a href="/change-data/change-address" target="_blank">update the registered office address (opens in a new tab)</a>
```

Comment:
i'd keep the rel="noopener noreferrer" data-event-id="update-roa-link" values in there as the prototype link won't have the matomo event. The rel="noopener noreferrer" is a security recommendation from Sonar i think.